plugins/linux/l3plugin/route_config.go

```diff
@@ -471,6 +494,19 @@ func (plugin *LinuxRouteConfigurator) ResolveDeletedInterface(name string, index
 	return nil
 }
 
+// RouteIdentifier generates unique route ID used in mapping
+func (plugin *LinuxRouteConfigurator) RouteIdentifier(route *netlink.Route) string {
+	var dstAddr string
+	if route.Dst != nil && route.Dst.IP != nil && route.Dst.Mask != nil {
+		maskSize, _ := route.Dst.Mask.Size()
+		dstAddr = route.Dst.IP.String() + "/" + strconv.Itoa(maskSize)
+	}
+	if dstAddr == ipv4AddrAny || dstAddr == ipv6AddrAny {
```

Comment:
This if condition doesn't make sense outside of the previous one since `dstAddr` will always be zero if the previous one is false.